src/answer/precious.rs

```diff
@@ -299,6 +321,17 @@ fn guess_syntax<'a>(possible_tags: &[String], ss: &'a SyntaxSet) -> &'a SyntaxRe
     ss.find_syntax_plain_text()
 }
 
+fn class_string_syntax<'a>(class_string: String) -> Option<String> {
```

Comment:
`<'a>` is not required.  And maybe take the class_string as reference is more better?
```rust
fn class_string_syntax(class_string: &str) -> Option<&str>
```

So we can just take something in `class_string` without creating a new `String` object